test/test_rake_functional.rb

```diff
@@ -295,6 +295,26 @@ def test_rules_chaining_to_file_task
            "'play.app' file should exist"
   end
 
+  def dryrun_tasks
+    @err.split("\n").select { |line|
+      line.match(/^\*\* Execute/)
+    }.map { |line|
+      line.gsub(/^\*\* Execute \(dry run\) /, "")
+    }
+  end
+
+  def test_update_midway_through_chaining_to_file_task
+    rakefile_file_chains
+
+    rake "-n"
+    assert_equal(%w{fileA fileB fileC default}, dryrun_tasks)
+    rake
+    sleep 1 # for stride seconds surely for timestamp
```

Comment:
Just a passer-by reading this comment. I'm confused by it. Can you reword it, to make it less ambiguous?

What I believe it wants to say: Ensure the timestamp is on a new second.